es6.js

```diff
@@ -224,6 +224,18 @@ var ES6 = assign(assign({}, ES5), {
 	// https://people.mozilla.org/~jorendorff/es6-draft.html#sec-samevaluezero
 	SameValueZero: function SameValueZero(x, y) {
 		return (x === y) || ($isNaN(x) && $isNaN(y));
+	},
+
+	// ES6 7.3.9 - http://www.ecma-international.org/ecma-262/6.0/#sec-getmethod
+	GetMethod: function GetMethod(obj, p) {
+		var func = obj[p];
```

Comment:
this lookup should be done with http://www.ecma-international.org/ecma-262/6.0/#sec-getv, and not directly with a property lookup